dask_cuda/benchmarks/utils.py

```diff
@@ -183,6 +188,8 @@ def setup_memory_pool(pool_size=None, disable_pool=False):
     import rmm
 
     rmm.reinitialize(
-        pool_allocator=not disable_pool, devices=0, initial_pool_size=pool_size,
+        pool_allocator=not disable_pool,
+        devices=0,
+        initial_pool_size=pool_size,
```

Comment:
I think we should change `--disable-rmm-pool` to `--enable-rmm-pool`, and we can have it enabled by default. The disable behavior is confusing, as I think most people would expect it not to do anything. IOW, I would expect something like:

```python
if enable_rmm_pool:
    rmm.reinitialize(...)
```

Reply:
Maybe we should take this out all together.  Who was using this feature ?  @madsbk or @rjzamora do you recall why this is here ?

Reply:
What feature are you talking about? Note that we have at least two different use cases:

1. Cluster created by the benchmark itself (with or without `--multi-node`): we want to set the pool;
1. Cluster started externally: we may _not_ want to set the pool.

Less important, that can be useful to test behavior if we think there may be an issue related to the pool.

Reply:
I'm talking about the flag which controls `pool_allocator=not disable_pool` .  When do we want rmm to be reinitialized without the pool ?  If we have the flag which controls the entire block of rmm I'm not sure we need the second flag

Reply:
In the current context, I don't think we should reinitialize unless we're setting a pool, thus the `if enable_rmm_pool` code above, we're not doing any modifications in that case so we don't need to reinitialize. Plus, we don't want to mess with the a cluster that's started externally, because of CUDA IPC not de-registering memory, we invalidate a cluster once we try to `rmm.reinitialize`, meaning the user would require to restart the cluster when running `local_cuda_merge.py` multiple times.